jupyterhub/templates/hub/configmap.yaml

```diff
@@ -11,6 +11,7 @@ data:
 {{- $_ := set $values "auth" (merge dict .Values.auth) }}
 {{- $_ := set $values.auth "state" (omit $values.auth.state "cryptoKey") }}
 {{- $_ := set $values "hub" (omit $values.hub "cookieSecret" "extraEnv" "extraConfigMap") -}}
+{{- $_ := set $values.hub "services" dict }}
 {{- range $key, $service := $values.hub.services }}
```

Comment:
You will now iterate over an empty dictionary

Reply:
It will be fine if we write...
```
{{- range $key, $service := .Values.hub.services }}
```
... I think. $key is a value type, and $service is a dict, but when we call `(omit $service "apiToken")` later, we will make a shallow copy I think.

Reply:
Good catch. Fixed.